### PR TITLE
tidb: fix config value for tiproxy

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

TiProxy is going to be released. This option is no more a pure warning.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions) 
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15635
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -439,19 +439,15 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 
 ### `session-token-signing-cert` <span class="version-mark">从 v6.4.0 版本开始引入</span>
 
-> **警告：**
->
-> 该配置与一个未发布的特性相关。**请勿设置该配置**。
-
++ 证书文件路径，用于 TiProxy 的会话迁移。
```
Comment (hfxsd) [inline tidb-configuration-file.md:442]: 这里是不是可以加给 TiProxy 的链接？

**Review by hfxsd — APPROVED** [review]

File: tidb-configuration-file.md (hunk 1 above)
Comment (xhebox) [reply tidb-configuration-file.md:442]: 应该可以链接到首页?

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -439,19 +439,15 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
 + 默认值：""
++ 空值将导致 TiProxy 会话迁移失败。要启用会话迁移，所有的 TiDB 节点必须设置相同的证书和密钥。因此你应该在每个 TiDB 节点上存储相同的证书和密钥。
 
 ### `session-token-signing-key` <span class="version-mark">从 v6.4.0 版本开始引入</span>
 
-> **警告：**
->
-> 该配置与一个未发布的特性相关。**请勿设置该配置**。
-
++ 密钥文件路径，用于 TiProxy 的会话迁移。
```
Comment (ran-huang) [inline tidb-configuration-file.md:448]:
```suggestion
+ 密钥文件路径，用于 [TiProxy](/tiproxy-overview.md) 的会话迁移。
```

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -439,19 +439,15 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 
 ### `session-token-signing-cert` <span class="version-mark">从 v6.4.0 版本开始引入</span>
 
-> **警告：**
->
-> 该配置与一个未发布的特性相关。**请勿设置该配置**。
-
++ 证书文件路径，用于 [TiProxy](/tiproxy-overview.md) 的会话迁移。
```
Comment (ran-huang) [inline tidb-configuration-file.md:442]:
```suggestion
+ 证书文件路径，用于 [TiProxy](/tiproxy/tiproxy-overview.md) 的会话迁移。
```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -439,19 +439,15 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
 + 默认值：""
++ 空值将导致 TiProxy 会话迁移失败。要启用会话迁移，所有的 TiDB 节点必须设置相同的证书和密钥。因此你应该在每个 TiDB 节点上存储相同的证书和密钥。
 
 ### `session-token-signing-key` <span class="version-mark">从 v6.4.0 版本开始引入</span>
 
-> **警告：**
->
-> 该配置与一个未发布的特性相关。**请勿设置该配置**。
-
++ 密钥文件路径，用于 [TiProxy](/tiproxy-overview.md) 的会话迁移。
```
Comment (ran-huang) [inline tidb-configuration-file.md:448]:
```suggestion
+ 密钥文件路径，用于 [TiProxy](/tiproxy/tiproxy-overview.md) 的会话迁移。
```

**Pull request merged**